kapitan/resources.py

```diff
@@ -99,6 +102,19 @@ def jinja2_render_file(search_paths, name, ctx):
 
     raise IOError("jinja2 failed to render, could not find file: {}".format(_full_path))
 
+def yaml_to_json(search_paths, name):
+    """return content of file in json"""
+    for path in search_paths:
+        _full_path = os.path.join(path, name)
+        logger.debug("yaml_to_json trying file %s", _full_path)
+        if os.path.exists(_full_path):
```

Comment:
It would also be a good idea to check if the file ends with yml or yaml here to avoid unnecessary exceptions.